pysmt/solvers/msat.py

```diff
@@ -47,6 +47,18 @@
 from pysmt.walkers.identitydag import IdentityDagWalker
 
 
+def _fix_var_name(n, back=False):
```

Comment:
Why can't we limit this to the Converter, in the transformation towards mathsat?